pkg/rpcmiddleware/discover/discover.go

```diff
@@ -74,11 +74,44 @@ func WithTransportCredentials(creds credentials.TransportCredentials) []grpc.Dia
 	}
 }
 
-// WithInsecure returns a DialOption which disables transport security and discovers the default insecure listen port if
-// not specified in the dial target.
+// WithInsecure returns gRPC dial options which disable transport security and discovers the default insecure listen
+// port if not specified in the dial target.
 func WithInsecure() []grpc.DialOption {
 	return []grpc.DialOption{
 		grpc.WithInsecure(),
 		grpc.WithContextDialer(resolver(false)),
 	}
 }
+
+type tlsFallbackKeyType struct{}
+
+var tlsFallbackKey tlsFallbackKeyType
+
+// WithTLSFallback returns a derived context which is configured to fall back to the given TLS setting if discovery
```

Comment:
Weird line break.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/blob/master/CONTRIBUTING.md#line-length, we're at 120

Reply:
How about breaking earlier then?
It just looks weird to see 1 word on a line, that's it.

Also `,` before `which`